src/renderRepoCard.js

```diff
@@ -52,19 +52,23 @@ const renderRepoCard = (repo, options = {}) => {
         <text data-testid="lang" class="gray" x="15">${langName}</text>
       </g>
 
-      <g transform="translate(${155 - shiftText}, 100)">
-        <svg class="icon" y="-12" viewBox="0 0 16 16" version="1.1" width="16" height="16">
-          ${icons.star}
-        </svg>
-        <text data-testid="stargazers" class="gray" x="25">${totalStars}</text>
-      </g>
+      ${stargazers.totalCount > 0 && `
```

Comment:
Using && would cause the rendered markup to have a `false` string in it.

![svg_false_pr](https://user-images.githubusercontent.com/35374649/87855598-0c999f00-c937-11ea-8ab3-e680fb752e6c.PNG)